version/src/test/java/io/smallrye/common/version/MavenVersionTest.java

```diff
@@ -49,20 +50,26 @@ public void testMavenCompare() {
         checkMavenConsistency("0_0", "0");
         checkMavenConsistency("0_0", "0_0final");
         // https://issues.apache.org/jira/browse/MNG-6964
-        // checkMavenConsistency("1-0.alpha", "1");
-        assertEquals(1, VersionScheme.MAVEN.compare("1-0.alpha", "1"));
+        checkMavenConsistency("1-0.alpha", "1");
         // https://issues.apache.org/jira/browse/MNG-6964
-        //checkMavenConsistency("1-0.beta", "1");
-        assertEquals(1, VersionScheme.MAVEN.compare("1-0.beta", "1"));
+        checkMavenConsistency("1-0.beta", "1");
+        // #267
+        checkMavenConsistency("3.6.0.CR1", "3.6.0");
+        checkMavenConsistency("3.6.0.FINAL", "3.6.0");
+        checkMavenConsistency("3.6.0.SP1", "3.6.0");
```

Comment:
Will `SP1` come before or after `3.6.0`?

Reply:
`CR1` comes before zero, `FINAL` is equal to zero, and `SP1` is greater than zero.